src/common/components/navigation/Menu.tsx

```diff
@@ -106,8 +106,8 @@ export const Menu: React.FunctionComponent<MenuProps> = (props) => {
   const [selectedIndex, setSelectedIndex] = useState(0);
 
   const handleSetSelectedIndex = () => {
-    const activeItem = menuItems.findIndex(
-      (item) => pathname === `${url}${item.path}`
+    const activeItem = menuItems.findIndex((item) =>
+      pathname.startsWith(`${url}${item.path}`)
```

Comment:
Just noticed that this solution sets `Trade` active in case `Trade History` is actually open.
```
pathname === `${url}${item.path}` || pathname.startsWith(`${url}${item.path}/`)
```
should fix this.